app/assets/stylesheets/components/autocomplete.scss

```diff
@@ -8,11 +8,13 @@
   }
 
   &__popover {
-    max-width: 300px;
     padding: var(--su-1);
     z-index: var(--z-elevate);
 
     &[data-reach-popover] {
+      // Reach UI adds a width directly to this element as an inline style. !important is needed to make sure the size is constrained.
+      width: 250px !important;
```

Comment:
Question (non-blocking) Just wondering if we want to keep the `max-width` as well. Also, just curious why we capped it at `300px` instead of some other value?

Reply:
I picked that width mostly because it matches our dropdown min width (and it just looked OK to me) 🙂   

I think it makes sense to keep it at one size (rather than a different min/max), as otherwise when different names get fetched/displayed, the width of the dropdown changes and it's a bit sketchy looking.